docs/rafter/01-01-rafter.md

```diff
@@ -21,11 +21,12 @@ Rafter comes with the following set of services and extensions compatible with R
 - [AsyncAPI Service](#details-asyncapi-service) (extension)
 - [Front Matter Service](#details-front-matter-service) (extension)
 
+>**CAUTION:** Rafter does not enforce any access control. To protect the confidentiality of your information, use Rafter only to store public data. Do not use it to process and store any kind of confidential information, including personal data.
 
 ## What Rafter is not
 
-* Rafter is not a [Content Management System](https://en.wikipedia.org/wiki/Content_management_system) (Wordpress-like),
-* Rafter is not a solution for [Enterprise Content Management](https://en.wikipedia.org/wiki/Enterprise_content_management),
+* Rafter is not a [Content Management System](https://en.wikipedia.org/wiki/Content_management_system), such as Wordpress-like.
```

Comment:
```suggestion
* Rafter is not a [Content Management System](https://en.wikipedia.org/wiki/Content_management_system), such as Wordpress.
```
or `Rafter is not a Wordpress-like [Content Management System](https://en.wikipedia.org/wiki/Content_management_system). `